Orange/base.py

```diff
@@ -443,7 +443,7 @@ def fix_dim(x):
         # Call the predictor
         backmappers = None
         n_values = []
-        if isinstance(data, (np.ndarray, scipy.sparse.csr_matrix)):
```

Comment:
Do you remember who calls __call__ directly with matrices and not with data? Did you need to make this change or did you do it for safety?

If it was me I'd deprecate this base functionality.

Reply:
I can't seem find why I put this in anymore. Guess I'll take it out for now and hope nothing breaks.